test/votingMachine.js

```diff
@@ -0,0 +1,35 @@
+const helpers = require('./helpers');
+const AbsoluteVote = artifacts.require('AbsoluteVote');
+const GenesisProtocol = artifacts.require('GenesisProtocol');
+const QuorumVote = artifacts.require('QuorumVote');
+const StandardToken = artifacts.require('StandardToken');
+const Reputation = artifacts.require('Reputation');
+const ExecutableTest = artifacts.require('ExecutableTest');
+const Avatar = artifacts.require('Avatar');
+
+contract('VotingMachine', (accounts)=>{
+  it('proposalId should be globally unique', async () =>{
+    const token = await StandardToken.new();
+    const rep = await Reputation.new();
+    const absolute = await AbsoluteVote.new();
+    const genesis = await GenesisProtocol.new(token.address);
+    const quorum = await QuorumVote.new(token.address);
+    const avatar = await Avatar.new('name', helpers.NULL_ADDRESS, rep.address);
+    const executable = await ExecutableTest.new();
+
+    const absoluteParams = await absolute.setParameters.call(rep.address,50,true);
+    await absolute.setParameters.sendTransaction(rep.address,50,true);
```

Comment:
why sendTransaction and not just setParameters ? 
it will be nice to keep consistency for with all other tests.

Reply:
when you do a `absolute.setParameters(..)` you let web3.js decide whether to `call` or to `sendTransaction`. In this case I think it's better to explicitly write that the first line only get's the result of the function and the second one actually changes the state of the contract.

Reply:
This is true if you are writing  raw requests via web3.js to the Ethereum network yourself.
Truffle is taking care of that (choosing between 'call' or  'sendTransaction' ) according to what the func is doing (write or read data).
I would ommit  'sendTransaction' for this case.

Reply:
updated: https://github.com/daostack/arc/pull/392/commits/ae904a0b4448b164af79ca781b211892d8d5cc98